app/workers/webhook_transfer.py

```diff
@@ -55,20 +55,24 @@ async def process_item_list(key: str) -> Optional[str]:
         retry = False
     while item_data := await redis.db.lpop(key):
         form_name, body = pickle.loads(item_data)
-        item = ANHash.from_parts(form_name, body)
         try:
-            if form_name == "donation":
-                prinl(f"Found donation item.")
-                await transfer_donation(item)
-            elif form_name == "upload":
-                prinl(f"Found upload item.")
-                await transfer_person(item)
+            if form_name == "shift":
+                prinl(f"Found shift item.")
+                await transfer_shift(body)
             else:
-                prinl(f"Found {form_name} submission item.")
-                await transfer_person(item)
+                item = ANHash.from_parts(form_name, body)
+                if form_name == "donation":
+                    prinl(f"Found donation item.")
+                    await transfer_donation(item)
+                elif form_name == "upload":
+                    prinl(f"Found upload item.")
+                else:
+                    prinl(f"Found {form_name} submission item.")
+                    await transfer_person(item)
             success_count += 1
             if env() is Environment.DEV:
                 logging_key = redis.get_key("Successfully processed")
+                print("logged??")
```

Comment:
Sorry, what?  Printing "logged??".  I think this should come out.

FYI, this `if` is just for integration with the test module, and I will take it out when I revamp the tests to be done correctly with mock Airtable interfaces (which I am working on now).  So if you want to take out this entire `if env() is Environment.DEV` block, I'm fine with that.